Microsoft.DotNet.Maestro.Tasks.Tests/AddAssetTests.cs

```diff
@@ -0,0 +1,64 @@
+using FluentAssertions;
+using Microsoft.DotNet.Maestro.Client.Models;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+
+namespace Microsoft.DotNet.Maestro.Tasks.Tests
+{
+    [TestFixture]
+    public class AddAssetTests
```

Comment:
In your Test doc you called out the following:

Given golden path with a combo of shipping and non-shipping assets
Given empty string parameters

This leaves me with a couple of questions?
- What constitutes a Shipping asset vs a non-shipping asset and which tests are covering this scenario?
- Are these all the tests or should we be adding empty parameter test case(s) to this review?

Reply:
Shipping vs non-shipping is just a bool in this case, it doesn't split the code path like it does in some of the methods (which is why it's called out in a few places), but just to make sure it's covered I did a positive test with one of each. I did plan to to swap the two in the two asset test though, so thanks for the reminder on that.

There's nothing in this method that would have a problem with the other arguments being null or empty, however I'm going to go through the flow and see if things would fall apart later (in which case we might to handle it here instead).

Reply:
@Chrisboh I'm not seeing anywhere that would have a problem with empty/null for the other values. They continue to be handled as strings all the way through. I'll add the additional tests if the need comes up, but at this point I don't think they would add any value.

Reply:
Sounds good.